src/rp2_common/hardware_i2c/include/hardware/i2c.h

```diff
@@ -246,6 +251,7 @@ static inline i2c_inst_t *i2c_get_instance(uint num) {
  *
  * \return Number of bytes written, or PICO_ERROR_GENERIC if address not acknowledged, no device present, or PICO_ERROR_TIMEOUT if a timeout occurred.
  */
+i2c_result_t i2c_write_result_blocking_until(i2c_inst_t *i2c, uint8_t addr, const uint8_t *src, size_t len, bool nostop, absolute_time_t until);
```

Comment:
Does this (and the other new `_result_` functions) need to be added to the header file _after_ the old functions, in order not to invalidate the documented doxygen `\return` parameter?